sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -2115,98 +2115,63 @@ def on_add_logical_port(self, port_change_event):
-                    if error_dict:
-                        vendor_specific_error_description = error_dict.get(port_change_event.port_index)
-                    else:
-                        vendor_specific_error_description = _wrapper_get_sfp_error_description(port_change_event.port_index)
-                    error_descriptions.append(vendor_specific_error_description)
-
-                error_description = '|'.join(error_descriptions)
-                helper_logger.log_info("Receive error update port sfp status table.")
-                if sfp_status_helper.is_error_block_eeprom_reading(error_bits):
-                    read_eeprom = False
-
-            # SFP information not in DB
-            if _wrapper_get_presence(port_change_event.port_index) and read_eeprom:
-                logical_port_event_dict[port_change_event.port_name] = sfp_status_helper.SFP_STATUS_INSERTED
-                transceiver_dict = {}
-                status = sfp_status_helper.SFP_STATUS_INSERTED if not status else status
-                rc = post_port_sfp_info_to_db(port_change_event.port_name, self.port_mapping, int_tbl, transceiver_dict)
-                if rc == SFP_EEPROM_NOT_READY:
-                    # Failed to read EEPROM, put it to retry set
-                    self.retry_eeprom_set.add(port_change_event.port_name)
+            if sfp_status_helper.has_vendor_specific_error(error_bits):
+                if error_dict:
+                    vendor_specific_error_description = error_dict.get(port_change_event.port_index)
                 else:
-                    post_port_dom_info_to_db(port_change_event.port_name, self.port_mapping, dom_tbl)
-                    post_port_dom_threshold_info_to_db(port_change_event.port_name, self.port_mapping, dom_threshold_tbl)
-                    update_port_transceiver_status_table_hw(port_change_event.port_name,
-                                                            self.port_mapping,
-                                                            status_tbl)
-                    post_port_pm_info_to_db(port_change_event.port_name, self.port_mapping, pm_tbl)
-                    notify_media_setting(port_change_event.port_name, transceiver_dict, self.xcvr_table_helper.get_app_port_tbl(port_change_event.asic_id), self.port_mapping)
+                    vendor_specific_error_description = _wrapper_get_sfp_error_description(port_change_event.port_index)
+                error_descriptions.append(vendor_specific_error_description)
+
+            error_description = '|'.join(error_descriptions)
+            helper_logger.log_info("Receive error update port sfp status table.")
+            if sfp_status_helper.is_error_block_eeprom_reading(error_bits):
+                read_eeprom = False
+
+        # SFP information not in DB
+        if _wrapper_get_presence(port_change_event.port_index) and read_eeprom:
+            logical_port_event_dict[port_change_event.port_name] = sfp_status_helper.SFP_STATUS_INSERTED
```

Comment:
where is logical_port_event_dict{} used?

Reply:
I saw it is no longer used, but this is not a change introduced by this PR. Anyway, I will remove it.